src/license.rs

```diff
@@ -4,21 +4,41 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+use dusk_bytes::Serializable;
 use dusk_jubjub::JubJubAffine;
 use dusk_jubjub::{dhke, GENERATOR_EXTENDED, GENERATOR_NUMS_EXTENDED};
+use dusk_merkle::poseidon::Tree;
+use dusk_merkle::{poseidon::Item, poseidon::Opening, Aggregate};
 use dusk_pki::{PublicKey, PublicSpendKey, SecretKey, SecretSpendKey, StealthAddress};
 use dusk_poseidon::cipher::PoseidonCipher;
 use dusk_poseidon::sponge;
 use dusk_schnorr::Signature;
 use rand_core::{CryptoRng, RngCore};
 
-use dusk_bytes::Serializable;
-
 use dusk_plonk::prelude::*;
-use dusk_poseidon::tree::PoseidonBranch;
 
 use crate::state::State;
 
+pub const DEPTH: usize = 17;
```

Comment:
Two things here:
1 - To comply with the wording that Moana and Ed proposed, and would say `HEIGHT` instead of `DEPTH`.
2 - These constants should be declared in `tests/citadel.rs` only, and propagated through the code.

Reply:
we decided to stick with DEPTH as it is not misleading compared to HEIGHT, which can have really two meanings: total height or current height. With depth we think less of current depth but rather always of total depth so this term conveys the actual meaning much better in our opinion